database/dynamodb.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"log"
+	"os"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/Real-Dev-Squad/feature-flag-backend/utils"
+)
+
+
+var Constants map[string]string  = map[string]string {
+	"ENV" : 				"ENVIRONMENT",
+	"DEV": 					"DEVELOPMENT",
+	"PROD":					"PRODUCTION",
+	"REGION": 				"AWS_REGION",
+	"ACCESS_KEY":			"AWS_ACCESS_KEY",
+	"SECRET_KEY":			"AWS_SECRET_KEY",
+}
```

Comment:
nit: 
Why is this inconsistently spaced out?

If it is done automatically by the go command or the go format you can resolve the comment and move ahead.

Reply:
run `go fmt` before you commit code or configure a pre-commit hook to do that for you.

Reply:
Sure